adoc/release-4.5.0.adoc

```diff
@@ -94,3 +106,11 @@ Fixes for these will be provided as a maintenance update.
 Note that the version of envoy shipped with CaaSP 4.5.0 contains security issues idendified as CVE-2020-12605,CVE-2020-8663,CVE-2020-12603 and CVE-2020-12604
 
 These are "Denial of Service" vulnerabilities, and do not expose systems to unauthorized access or data exfiltration. A fix for them will be provided as a maintenance update.
+
+
+=== Helm 2to3 migration plugin requires internet connection to install
+
+The installer for the Helm 2to3 plugin is written to pull the plugin from the official community github site at link:https://github.com/helm/helm-2to3[github.com/helm/helm-2to3].
+This could cause a problem in an air-gapped {productname} installation where an open internet connection is not available.
+
+The simplest workaround is to move the management system (such as a laptop) out of the internal network, install the plugin, then move back in to perform the migration.
```

Comment:
I think this is sufficient, as moving in and out of the 'internal' network is something the user will do anyway to pick up updated containers and packages.